.github/workflows/duck.yml

```diff
@@ -26,11 +26,11 @@ jobs:
             type=semver,pattern={{major}}.{{minor}}.{{patch}},value=${{env.VERSION}}
             type=semver,pattern={{major}}.{{minor}},value=${{env.VERSION}}
             type=semver,pattern={{major}},value=${{env.VERSION}}
-      - name: Set up JDK 15
+      - name: Set up JDK 17
         uses: actions/setup-java@v2
         with:
-          distribution: 'zulu'
-          java-version: 15
+          distribution: 'microsoft'
```

Comment:
Switch to `temurin` as this is the OpenJDK build we use on macOS for distribution.